include/cling/Interpreter/Interpreter.h

```diff
@@ -712,6 +713,24 @@ namespace cling {
     ///
     void* getAddressOfGlobal(llvm::StringRef SymName, bool* fromJIT = 0) const;
 
+    ///\brief Get a given macro definition by name.
+    ///
+    ///\param[in]  Name - the name of the macro to look for
+    ///
+    ///\returns the MacroInfo if the macro was defined, otherwise null
+    ///
+    const clang::MacroInfo* getMacro(llvm::StringRef Name) const;
```

Comment:
> any future introduction 
It was a description of the current usage of the word in ROOT (see for example https://root.cern.ch/doc/v608/TMacro_8h_source.html)

Now on the other hand, you are right, it is indeed right it already scoped by being inside cling::Interpreter.  So fair enough, let's leave it.

thanks,
Philippe